opt_PXL.ino

```diff
@@ -283,13 +297,12 @@ void setupLEDs() {
 }
 
 void setupOLED(){
-  Serial.begin(9600);
 
   // by default, we'll generate the high voltage from the 3.3v line internally! (neat!)
   display.begin(SSD1306_SWITCHCAPVCC, 0x3C);  // initialize with the I2C addr 0x3D (for the 128x64)
   // init done
   display.display();     // show splashscreen
-  delay(2000);
+  delay(1000);

```

Comment:
The amount of magic numbers is too damn high! :joy: :trollface:

Reply:
These are just delays so. That entire function is required to setup the OLED where it shows the adafruit industries splash screen `display.display();` then `delay(1000); //wait one second` then carry on with the rest of the code.